scanner/Makefile

```diff
@@ -72,6 +70,27 @@ endif
 tag:
 	@echo $(TAG)
 
+###############
+## Go Builds ##
+###############
+
+build-cmd := $(BUILD_CMD)
+build-d   := bin
+build-t   := $(addprefix $(build-d)/,$(notdir $(wildcard cmd/*)))
+
+.PHONY: build
+build: $(build-t)
+
+.PHONY: $(build-t)
+$(build-t): deps
+	@echo "+ $@"
+	$(SILENT)$(build-cmd) -o $@ ./cmd/$(@F)
```

Comment:
Why silent here? I think it could be helpful to see the build command

Reply:
> Why silent here? 

The purpose of $(SILENCE) is to hide the details of the rules unless `SILENT=`.